pkg/arvo/sys/vane/clay.hoon

```diff
@@ -1142,9 +1150,10 @@
   ++  aver
     |=  [for=(unit ship) mun=mood]
     ^-  [(unit (unit cage)) ford-cache]
-    =+  ezy=?~(ref ~ (~(get by haw.u.ref) mun))
-    ?^  ezy
-      [`u.ezy fod.dom.red]
+    ::  XX: wat do
```

Comment:
You are going to want to look up the request in `haw`.  IIRC this is how non-full-sync foreign requests are filled.